### PR TITLE
Several small changes to clean up code

### PR DESCRIPTION
**Pull request by david-rosales**
I followed the suggestions left in the previous pull request and implemented the fixes. This includes no longer clearing every time something is saved to `localStorage`, using `bind` instead of creating an `instance` variable, renaming the function props, and using `_.maxBy` instead of a for loop.

### CODE REVIEW

**Comment by david-rosales** [comment]
@kevinrobinson I implemented the changes you suggested, can you review it?

File: src/informationcards.jsx [hunk 1]
```diff
@@ -148,14 +148,9 @@ var EditableCard = React.createClass({
...
-    if(cards !== null){
-        for(var x=0;x<cards.length;x++){
-            var card = cards[x];
-            if(card.id > largestId){
-                largestId = card.id;
-            }
-        }
-        largestId+=2;
+    if(cards !== null && cards.length > 0){
+        var mostRecentCard = _.maxBy(cards, function(card){return card.id;});
```
Comment (kevinrobinson) [inline src/informationcards.jsx]: Nice improvement!  You can even use the `_.maxBy` shorthand if you like: https://lodash.com/docs#maxBy

File: informationcards.html [hunk 2]
```diff
@@ -8,6 +8,7 @@
         <script src="https://fb.me/react-dom-15.1.0.js"></script>
         <script src="https://cdnjs.cloudflare.com/ajax/libs/babel-core/5.8.23/browser.min.js"></script>
         <script src="https://cdn.jsdelivr.net/lodash/4.13.1/lodash.min.js"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/velocity/1.2.3/velocity.js"></script>
```
Comment (kevinrobinson) [inline informationcards.html]: This looks like subsequent work on animation?  It's cool to merge that in now if you're working on it, just checking.

**Comment by kevinrobinson** [comment]
image

**Pull request merged**